### PR TITLE
Replaced Toasts with Snackbar

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Replaced all the toasts with snackbars except in Service class.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

File: .idea/Travel-Mate.iml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Swati4star) [inline .idea/Travel-Mate.iml]: This & next 3 xml files should not be a part of PR

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java [hunk 2]
```diff
@@ -216,7 +215,9 @@ public void onResponse(Call call, final Response response) {
                         lv.setAdapter(new CityAdapter(mActivity, cities, settings));
                         lv.setOnItemClickListener((parent, view, position, id1) -> {
                             City city = (City) lv.getAdapter().getItem(position);
-                            Toast.makeText(mActivity, city.getNickname(), Toast.LENGTH_SHORT).show();
+                            Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java]: These shouldn't be a part of this PR

File: Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java [hunk 3]
```diff
@@ -160,8 +163,9 @@ public void startMainActivity() {
 
     @Override
     public void showError() {
-        Toast.makeText(this, R.string.toast_invalid_username_or_password, Toast.LENGTH_LONG)
-                .show();
+        Snackbar.make(Objects.requireNonNull(LoginActivity.this).findViewById(android.R.id.content),
+                R.string.toast_invalid_username_or_password,
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java]: change the name of strings in strings.xml also from toast to snackbar

File: Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java [hunk 4]
```diff
@@ -93,10 +97,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
         if (ACTIVITY_INSERT_CONTACT == requestCode) {
             if (resultCode == Activity.RESULT_OK) {
-                Toast.makeText(this, "Added Contact", Toast.LENGTH_SHORT).show();
+                Snackbar.make(Objects.requireNonNull(ShareContact.this).findViewById(android.R.id.content),
+                        "Added Contact",
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java]: Move the constant strings from here to strings.xml file

**Comment by shiv07tiwari** [comment]
Okay, working on the changes.

**Pull request closed without merging**